source/posts/2015-06-05-getting-started-with-ios-development.markdown

```diff
@@ -0,0 +1,102 @@
+---
+title: Getting Started with iOS Development
+category: iOS
+summary: I've recently started learning iOS and Swift and I wanted to share my experiences so far.
+date: 2015-06-05
+tags: swift
+---
+
+Around April I started digging into iOS and Swift. I figured it would
+be helpful to share my experience with others who may also want to learn it.
+Note, this comes from the perspective of someone who already has a programming
+background.  I had also previously started to learn iOS a few years ago. I spent
+a bit of time becoming familiar with Objective-C and Apple's Cocoa Touch
+frameworks.
+
+When I first got started I went searching around for some online iOS courses. I
+was hoping to find some courses on Code School or something similar.
+Unfortunately, I couldn't find any course like this. The courses that I found
+were focused around learning iOS with Objective-C. I decided if I was going to
+start learning iOS that I wanted to learn it with the modern Swift language
+rather than Objective-C.
+
+That's not to say that having Objective-C knowledge isn't valuable. It's
+especially important to have Objective-C experience to work with older
+applications. For example, [Tropos](http://troposweather.com/) is built with
+Objective-C. In order to contribute to this project, I need to think back to my
+Objective-C learnings. The specifics around iOS development tend to be the
+frameworks that Apple provides to its developers. These frameworks have great
+documentation and are documented in both, Swift and Objective-C. If you're
+learning how to use these libraries and the concepts behind iOS, it shouldn't be
+difficult to apply them to Objective-C.
+
+I was disappointed to see that there weren't many resources to get started in
+iOS from a beginner's perspective with Swift. After spending some more time
+searching I found that Stanford University has a free course on iTunes
+university titled [Developing iOS 8 Apps with
+Swift](https://itunes.apple.com/us/course/developing-ios-8-apps-swift/id961180099).
+This has been an immensely useful resource. Paul Hegarty is a great instructor
```

Comment:
hard stop after " Paul Hegarty is a great instructor." " He's really good at explaining the concepts and demonstrating things in real time."